google/cloud/dialogflow_cx/doc/main.dox

```diff
@@ -1,9 +1,12 @@
 /*!
 
-@mainpage Dialogflow API C++ Client Library
+@mainpage Dialogflow CX API C++ Client Library
 
 An idiomatic C++ client library for the [Dialogflow API][cloud-service-docs], a service
-to Builds conversational interfaces (for example, chatbots, and voice-powered apps and devices).
+to build conversational interfaces (for example, chatbots, and voice-powered apps and devices).
+Dialogflow supports two [editions], this library supports the CX edition.
```

Comment:
Same "There are [two editions] of Dialogflow. This library supports the CX edition." comment as above.

Reply:
Fixed.